draft/spec/index.html

```diff
@@ -414,10 +414,10 @@ <h2>Digests</h2>
             through hardware degradation, accident, or malicious actors.
         </p>
         <p>
-            For content-addressing, OCFL Objects MUST use either <code>sha512</code> or <code>sha256</code> 
-            digests, and SHOULD use <code>sha512</code> digests. The choice of the <code>sha512</code> digest
-            algorithm recognizes that it has no known collision vulnerabilities and is less computationally
-            intensive to compute than <code>sha256</code>, [[Stop-Using-SHA-256]].
+            For content-addressing, OCFL Objects SHOULD use <code>sha512</code>. The choice of the <code>sha512</code>
+            digest algorithm as default recognizes that it is widely available in different implementations, has no
```

Comment:
I agree with @awoods that it would be better without the added _"widely available in different implementations,"_. If others feel that we need to say something about availability then I think we need to improve the wording.

Reply:
The intention was to communicate that it's a safe choice -- not exotic, implemented in several languages and environments, etc. As opposed to, say SHA-3.

Reply:
I don't think that is necessary

Reply:
.. and the reason I think that may be worthwhile to include is because there is an expectation that the defaults for OCFL are 'best-practice-worthy' for digital preservation. If you don't know MD5 from SHA-512, you may want to both have the recommendation (SHOULD) but also a brief paragraph about why. Otherwise it seems like it is an arbitrary decision.

Reply:
If we were to add something then it should be the last item in the list of reasons (being fit for purpose is more important) and we need to make sure that there isn't ambiguity about which implementations we are talking about (OCFL or the algorithm). Could be something like _"..., and multiple implementations are available."_ However, I'm still not sure it is needed. Other input would be good...

Reply:
... also, wide availability of the hashing algorithm was a factor in our choosing of it. BLAKE2b is superior, but it's not as widely implemented.

Reply:
Language could go into issue #186 and I can make the update.

Reply:
Editors call: Decision to change to ", and multiple implementations are available".